.github/CODEOWNERS

```diff
@@ -276,6 +276,9 @@
 /pkg/network/ebpf/c/prebuilt/http*      @DataDog/universal-service-monitoring
 /pkg/network/ebpf/c/runtime/http*       @DataDog/universal-service-monitoring
 /pkg/network/ebpf/c/http*               @DataDog/universal-service-monitoring
+/pkg/network/ebpf/c/go-tls-*            @DataDog/universal-service-monitoring
+/pkg/network/ebpf/c/tags-types.h        @DataDog/universal-service-monitoring
+/pkg/network/ebpf/go_tls*              @DataDog/universal-service-monitoring
```

Comment:
Wrong formatting here.

Reply:
Fixed in [09fd23a](https://github.com/DataDog/datadog-agent/pull/13852/commits/09fd23acc9ef1afaedbf3a8edbdbe4aa53f9f075)